unittests/src/main/java/uk/ac/manchester/tornado/unittests/tools/TornadoHelper.java

```diff
@@ -162,13 +163,20 @@ static void runTestVerbose(String klassName, String methodName) throws ClassNotF
                     continue;
                 }
 
-                if (result.getFailures().stream().filter(e -> (e.getException() instanceof PTXNotSupported)).count() > 0) {
+                if (result.getFailures().stream().anyMatch(e -> (e.getException() instanceof PTXNotSupported))) {
```

Comment:
Let's also update the `PTXNotSupported` -> `TornadoPTXNotSupported`

Reply:
I think it is better to change `Tornado` with `TornadoVM` for every new reference that we add. What do you think @jjfumero?